setup.py

```diff
@@ -106,7 +106,7 @@ def main():
         pjoin('workflows', 'data', '*'),
         pjoin('pipeline', 'engine', 'report_template.html'),
         pjoin('external', 'd3.js'),
-        pjoin('interfaces', 'script_templates', '*'),
+        pjoin('script_templates', '*'),
```

Comment:
since the script_templates are all fsl model related, perhaps this can be moved into `interfaces/fsl/model/script_templates`